stellargraph/datasets/dataset_loader.py

```diff
@@ -0,0 +1,218 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019-2020 Data61, CSIRO
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Enable easy loading of sample datasets for demonstrations
+"""
+
+import os
+import logging
+from shutil import unpack_archive, move
+from urllib.request import urlretrieve
+from typing import List, Optional, Any
+from urllib.error import URLError
+
+
+log = logging.getLogger(__name__)
+
+
+class DatasetLoader(object):
```

Comment:
I don't think this is necessary in Python 3?

```suggestion
class DatasetLoader:
```